doc/build.md

```diff
@@ -55,71 +58,85 @@ Orientation does not matter. Solder based on the labeled values.
 | R2, R5 | 5.1k  |
 | R3, R4 | 75    |
 | R6     | 10k   |
-![Resistors](https://i.imgur.com/sZXMwDM.jpg)
 
-### Step 3. 1n1418 Diodes (D1-68, D71-74)
+![Resistors](https://imgur.com/haA16fU.png)
+
+### Step 3. 1n1418 Diodes (D1-68, D72-74)
 **This part has a specific orientation** - black line on the diode lines up with the square pad (points up)
-![Diodes](https://i.imgur.com/tWNR7mo.jpg)
+![Diodes](https://imgur.com/qEMczrJ.png)
+Don't forget the D72-74 ones !
```

Comment:
Maybe a comment for the left side macro cluster as well

Reply:
The left side macro cluster doesn't need additional diodes

Reply:
I didn't know, thanks!